Core/include/Acts/TrackFinding/MeasurementSelector.hpp

```diff
@@ -93,11 +95,15 @@ class MeasurementSelector {
          bool& isOutlier, const Logger& logger) const;
 
  private:
-  template <typename traj_t, typename cut_value_t>
-  static cut_value_t getCut(
-      const typename traj_t::TrackStateProxy& trackState,
-      const Acts::MeasurementSelector::Config::Iterator selector,
-      const std::vector<cut_value_t>& cuts, const Logger& logger);
+  struct Cuts {
+    double chi2Measurement{};
+    double chi2Outlier{};
+    std::size_t numMeasurements{};
+  };
+
+  static Cuts getCutsByTheta(const MeasurementSelectorCuts& config,
+                             double theta);
+  Result<Cuts> getCuts(const GeometryIdentifier& geoID, double theta) const;
```

Comment:
I wonder if these would be useful for user `MeasurementSelector`s (@goetzgaycken), now they support a separate outlier cut. It would be good to harmonise where possible.

Maybe best left to a separate PR, but worth considering soon.

Reply:
I agree this is just an intermediate step to get an outlier chi2 threshold that we can test in ACTS. The next step would be to generalize the measurement selector which requires breaking changes